File: java/java-tests/testData/findSimilar/Anonymous.java
class A {
  public int foo(A a) {
    return 0;
  }

  public static void main(String[] args) {
    A a1 = new A();
    int a = a1.f<caret>oo(new A() {
      public int foo(A a) { return 1; }
    });
  }
}

File: plugins/kotlin/idea/tests/testData/quickfix/contextParameter/addExplicitContextArguments/anonymous.kt
// "Add explicit context argument" "false"
// COMPILER_ARGUMENTS: -Xcontext-parameters -Xexplicit-context-arguments
// DISABLE_K2_ERRORS
context(_: String)
fun ctxFun(value: Int) {}

fun useAC3() {
    <caret>ctxFun(0)
}

// FUS_K2_QUICKFIX_NAME: org.jetbrains.kotlin.idea.k2.codeinsight.fixes.AddExplicitContextArgumentFix